src/runtime/hexagon/hexagon_device_api.h

```diff
@@ -138,7 +157,11 @@ class HexagonDeviceAPI final : public DeviceAPI {
   }
 
   //! \brief Manages underlying HexagonBuffer allocations
+  // runtime_hexbuffs is used for runtime allocations.  It is created
+  // with a call to AcquireResources, and destroyed on ReleaseResources.
```

Comment:
could you add some explanation of why the extra, non-RAII behavior is taken here?

Reply:
This is so we can ensure that resources are cleaned up for a runtime session, when we are doing tuning in a loop.  We want the client side to explicitly free resources.